contrib/mpi-proxy-split/mpi-wrappers/Makefile

```diff
@@ -32,6 +32,12 @@ override CXXFLAGS += -g3 -O0 -fPIC -I${DMTCP_INCLUDE} -I${JALIB_INCLUDE} \
                      -I${PLUGIN_HALF_SRCDIR} -I${DMTCP_ROOT}/src \
                      -I${LOWER_HALF_SRCDIR} -std=c++11
 
+ifeq ($(shell which python),python)
+  PYTHON = python
+else
+  PYTHON = python2.7
```

Comment:
Have you tried `python3`?
I think python2 is depreciated, we should think about upgrading the script if we have problems with python3.

And here, if we cannot work with python3, then we will have trouble if `python` is linked to `python3`

Reply:
`python3` is indeed working. i can change the script to use that instead and remove the python part from the tutorial entirely.